app/assets/stylesheets/views/_organisations.scss

```diff
@@ -1,24 +1,50 @@
 .organisations {
-  .organisations__section {
+  .organisations-list__section {
     margin-bottom: $gutter * 2;
 
     .department-count {
       @include bold-80;
     }
 
-    .organisations__logo-wrapper {
+    .organisations-list__item {
```

Comment:
Maybe outside the scope of this PR, but something we might want to address is that the clickable area of the logo extends all the way to the right. I've accidentally clicked on it a few times trying to open the 'view all' link. Maybe if it was inline-block? Would be a change to the component, I think.

<img width="567" alt="screen shot 2018-06-08 at 14 18 44" src="https://user-images.githubusercontent.com/861310/41160280-119706f2-6b27-11e8-827d-2bac150ceeb0.png">